README.md

```diff
@@ -1,6 +1,11 @@
 # Spacecraft
 
 ![image](https://travis-ci.org/pyymenta/spacecraft-cli.svg?branch=master)
+![contributors](https://img.shields.io/github/contributors/pyymenta/spacecraft-cli)
+![issues](https://img.shields.io/github/issues/pyymenta/spacecraft-cli)
+![forks](https://img.shields.io/github/forks/pyymenta/spacecraft-cli)
+![stars](https://img.shields.io/github/stars/pyymenta/spacecraft-cli)
```

Comment:
these I believe it's not necessary because this infos are easily finded in the project..

I believe you can keep the hacktoberfest badge and, if you could, help us to fix the license badge

Reply:
This issue aren't resolved yet